.github/workflows/linter.yml

```diff
@@ -45,3 +45,4 @@ jobs:
           flake8_auto_fix: false
           flake8_args: ${{ steps.changed-files.outputs.all_changed_files }}
           continue_on_error: false
+
```

Comment:
I don't wanna know how this solves the problem :see_no_evil: 
must've been frustrating to figure out